language/snippets/classify_text/requirements.txt

```
@@ -1,4 +1,4 @@
 google-cloud-language==2.15.1
-numpy==2.0.0; python_version > '3.9'
-numpy==1.24.4; python_version == '3.8'
-numpy==1.26.4; python_version == '3.9'
+numpy==2.2.4; python_version > '3.9'
+numpy==2.2.4; python_version == '3.8'
+numpy==2.2.4; python_version == '3.9'
```

```suggestion
```suggestion
numpy==2.2.4; python_version > '3.9'
numpy==1.26.4; python_version == '3.9'
numpy==1.24.4; python_version == '3.8'
```
```